src/macros.rs

```diff
@@ -211,7 +147,7 @@ macro_rules! intrinsics {
         $($rest:tt)*
     ) => (
         #[cfg($name = "optimized-c")]
-        #[cfg_attr(feature = "weak-intrinsics", linkage = "weak")]
+        #[cfg_attr(all(not(windows), not(target_vendor = "apple")), linkage = "weak")]
```

Comment:
This is not needed here: weak linkage is only needed for functions with `#[no_mangle]`.